src/components/predictive-layer.vue

```diff
@@ -14,8 +14,22 @@
           : 'x-row x-row--gap-06 x-row--align-start'
       "
     >
+      <IdentifierResults
+        #default="{ identifierResult }"
+        :animation="suggestionsAnimation"
+        class="x-row__item x-row__item--span-12 x-list x-list--gap-04"
+      >
+        <BaseResultLink #default="{ result }" :result="identifierResult" class="x-suggestion">
+          <BarCodeIcon />
+          <IdentifierResult class="x-text x-text--bold" :result="result" />
```

Comment:
I would limit the results to a maximum of 5, like in the query suggestions.

Reply:
Me too but @tajespasarela prefers to keep like it is right now.

Reply:
let's leave this way at the moment. The SKU search is a special case, to find a result or a set of them. If we "cut" that list, maybe the searched result is out of that.

We can tweak it in each customer setup, but I would leave as it is by now.